seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -765,7 +765,7 @@
       "title": "Nest an Anchor Element within a Paragraph",
       "description": [
         "Again, here's a diagram of an <code>a</code> element for your reference:",
-        "<a href=\"//i.imgur.com/hviuZwe.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
+        "<a target=\"_blank\" href=\"//i.imgur.com/hviuZwe.png\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
```

Comment:
@rarmatei Thanks for the PR! :smile: :tada:

Other contributors will have to give their opinions on this, but I think we should add `target="_blank"` together with `rel="noopener noreferrer"`. See [this blog post](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/) for reference.

I'd say this could be done for other challenges as well :blush: